Bylaws.md

```diff
@@ -3,108 +3,132 @@
+			<li>To maintain a modular, documented, open platform for machine translation and other human language processing tasks.</li>
 			<li>To favour the interchange and reuse of existing linguistic data.</li>
 			<li>To make integration with other free/open-source technologies easier.</li>
-			<li>To radically guarantee the reproducibility of machine translation and natural language processing research</li>
+			<li>To radically guarantee the reproducibility of machine translation and natural language processing research.</li>
 		</ol>
 	</li>
 </ol>
 
-## Roles and responsibilities
+## Roles and Responsibilities
 
 ### Users
 
 <ol start="2">
-	<li>The most important participants in the project are people who use Apertium. The majority of our developers start out as users and guide their development efforts from the user's perspective</li>
-	<li>Users contribute to the Apertium project by providing feedback to developers in the form of bug reports and feature suggestions. As well, users participate in the Apertium community by helping other users.</li>
+	<li>The most important participants are people who use Apertium. The majority of our contributors start out as users and guide their development efforts from the user's perspective.</li>
+	<li>Users contribute to Apertium by providing feedback to developers in the form of bug reports and feature suggestions. As well, users participate in the Apertium community by helping other users.</li>
 </ol>
 
-### Developers
+### Contributors and Contributions
 
 <ol start="4">
-	<li>Developers are  volunteers who are contributing time, code, documentation, or resources to Apertium. A developer that makes sustained, welcomed contributions to the project may be invited to become a committer, though the exact timing of such invitations depends on many factors</li>
+	<li>Contributors are volunteers who contribute time, code, documentation, or resources to Apertium. A contributor that makes sustained, welcomed contributions to Apertium may be invited to become a Voting Member, though the exact timing of such invitations depends on many factors.</li>
+	<li>Contributors who have administrative rights to a repository may exercise those rights almost fully, including granting rights to new contributors, making releases, or setting up automation. Potentially destructive or distruptive actions, such as deleting or renaming the repository or master branch, or breaking backwards compatibility, must be approved by a PMC member, who may decide the action requires a PMC vote.</li>
+	<li>Apertium recommends, but does not require, that contributors sign the Apertium Open Source Contributor License Agreement. However, contributions that are not licensed to Apertium in this way must be clearly marked with a GPLv2+ compatible license, plus the author's name and email or other identifying information sufficient for a lawyer to reach the contributor.</li>
```

Comment:
Free/Open-Source ? How about other contributions under CC-BY(-SA)?

Reply:
CC-BY is directly GPLv2+ compatible. CC-BY-SA 4.0 is GPLv3+ compatible, so that falls under the +. Too much of our other stuff is GPLv2+ and almost certainly will not be relicensed under anything else, so I see no other solution than to make sure anything new is up front compatible with GPLv2+. But we only need compatible - and many licenses are compatible. It is a very minor restriction.
What I would really want is copyright assignment from everyone, but I think that will be impossible to push through.

Reply:
Works for me, how about calling it Free/Open-Source though? @mlforcada what do you think?

Reply:
Oh, you want to change the name of the CLA? Sure. Doesn't matter what it is called - it's the clauses in it that matter.